src/Kentico.Kontent.ModelGenerator.Core/TypeProviderCodeGenerator.cs

```diff
@@ -82,6 +92,9 @@ public string GetCodename(Type contentType)
 
             var cu = (CompilationUnitSyntax)tree.GetRoot();
 
+            var description = SyntaxFactory.Comment(TypeProviderCodeGenerator.FILE_COMMENT_PREFIX + Environment.NewLine + Environment.NewLine);
```

Comment:
there might be useful to create a helper method for generating a comment

Reply:
Added new file SyntaxHelpers.cs with helper method GenerateComment(string), thats used in BaseClassGenerator, ClassCodeGenerator & TypeProviderCodeGenerator